lib/components/overlays/quick_settings/pages/qs_language_page.dart

```diff
@@ -42,24 +42,39 @@ class QsLanguagePage extends StatelessWidget {
       body: Padding(
         padding: const EdgeInsets.symmetric(vertical: 16.0),
         child: ListView.builder(
-          itemCount: Locales.supported.length,
+          itemCount: locales.supportedLocales.length,
           itemBuilder: (context, index) {
             return ListTile(
               shape: RoundedRectangleBorder(
                 borderRadius: BorderRadius.circular(8),
               ),
-              leading: Flag.fromString(
-                Locales.supported[index].languageCode.replaceAll("en", "us"),
-                width: 32,
-                replacement: const Icon(Icons.language_outlined),
+              leading: Text(
+                locales.supportedLocales[index]
+                    .toString()
+                    .substring(3)
+                    .replaceAllMapped(
+                      RegExp('[A-Z]'),
+                      (match) => String.fromCharCode(
+                        match.group(0)!.codeUnitAt(0) + 127397,
+                      ),
+                    ),
```

Comment:
Add a comment maybe on how this works? For example, what is the magic number 127397 and where did you get it? Also that substring(3), what's the reason on having that?